Vagrantfile

```diff
@@ -0,0 +1,48 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/xenial64"
+
+  config.vm.provision "shell", inline: <<-SHELL
+
+    echo "Provisioning Start"
+    export DEBIAN_FRONTEND=noninteractive
+    apt-get -qq update && apt-get -qq dist-upgrade
+
+    echo "Installing MySql"
+    sudo -E apt-get -qq -y install mysql-server
+    /etc/init.d/mysql stop
+    mysqld --skip-grant-tables &
+    mysql -u root -e "UPDATE user SET Password=PASSWORD('pass123') WHERE User='root'; FLUSH PRIVILEGES; exit;"
```

Comment:
Wouldn't it be better to interactively set the password, or leave it empty (so it wouldn't work) so that whoever is setting up the environment *has* to set the password before they can do so?